packages/babel-plugin/__tests__/legacy/transform-logical-properties-test.js

```diff
@@ -632,170 +616,6 @@ describe('@stylexjs/babel-plugin', () => {
       `);
     });
 
-    /**
-     * Non-standard properties
```

Comment:
These got moved to the 'polyfills' test file. But the goal is to be able to delete these once there is appropriate compiler validation.